pkg/client/client.go

```diff
@@ -205,6 +212,13 @@ func (cli *Client) SetSecure() error {
 	return nil
 }
 
+// UseCookies indicates session based authentication approach and cookies are used to avoid creating new sessions
```

Comment:
@leslie-qiwa , please remove the long description 1-2 lines up to 120 char each is enough.